backend/library/libraries/map-DNSSI_2023-iso27001_2022.yml

```
@@ -0,0 +1,839 @@
+urn: urn:intuitem:risk:library:map-dnssi-iso27001-2022
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)